docs/source/internals/getting_started.rst

```diff
@@ -186,6 +186,28 @@ you will also need to include Django's i18n URLs:
         url(r'', include(application.urls)),
     ]
 
+For `Django > 1.9`_ pass ``application.urls`` directly to ``path``. For `Django > 2.0`_ use ``path`` and ``re_path`` instead of ``url``.:
+
+.. _`Django > 2.0`: https://docs.djangoproject.com/en/2.0/releases/2.0/#simplified-url-routing-syntax
+.. _`Django > 1.9`: https://docs.djangoproject.com/en/2.0/releases/1.9/#passing-a-3-tuple-or-an-app-name-to-include
```

Comment:
We don't support Django 1.10 and below, so this comment is not necessary. Actually we need to drop all uses of `include()` with 3-tuples because it is deprecated in Django 1.11.

There are a few other places in the documentation that need to be updated to stop using `include()`: `how_to_change_a_url.rst`, `how_to_disable_an_app_or_feature.rst`.